### PR TITLE
conc-builds test 4.15

### PR DESCRIPTION
**Pull request by lenahorsley**

### CODE REVIEW

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-conc-builds-3nodes

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -4978,6 +4978,88 @@ periodics:
...
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: rosa-4.15-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-conc-builds-3nodes
+  spec:
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:5000]: lets add this black before spec here, no need to rerun after the change. will merge after this gets added :)

```
reporter_config:
    slack:
      channel: '#ocp-qe-scale-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :warning: {{end}}'
```

**Pull request merged**